src/express-tooling.ts

```diff
@@ -33,37 +34,41 @@ const AUTHORIZATION_HEADER_FIELD_NAME = 'authorization';
  *  app.get('/path', requireScopesMiddleware['scopeA', 'scopeB'], (req, res) => { // Do route work })
  *
  * @param scopes - array of scopes that are needed to access the endpoint
+ * @param logger - optional logger
```

Comment:
the typing specifies the "optional" status already, maybe instead we describe what is logged?

Reply:
The precedenceOptions is marked optional in the description as well - therefore we should be consistent. I am not sure if we should add what we log, as the logger is also passed to other functions. Furthermore we should think about additional logging besides error/warning

Reply:
I like the idea of having debug logs as well!